freqtrade/rpc/rest_client.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
```

Comment:
The rest-client can be a class or a set of methods that send commands to the http-server.